onnxscript/optimizer/_constant_folding.py

```diff
@@ -547,6 +547,46 @@ def sequence_construct(node: ir.Node, op, state: OptimizerState) -> ReturnValue:
     return None
 
 
+@register("Split")
+def split(node: ir.Node, op, _):
+    """Replaces Split operators with all constant inputs by a list of Constant operators."""
+    # Replace single output split by Identity(x)
+    if len(node.outputs) == 1:
+        return op.Identity(node.inputs[0])
+
+    # Skip non-constant inputs
+    if (x := ir.convenience.get_const_tensor(node.inputs[0])) is None:
+        return None
+
+    _split = None
```

Comment:
```suggestion
    split_ = None
```

tiny nit: prefixed `_` variables are typically unused. But not a big deal either